pytest_doctestplus/plugin.py

```diff
@@ -164,24 +163,23 @@ def collect(self):
             if self.fspath.basename == "setup.py":
                 return
             elif self.fspath.basename == "conftest.py":
-                try:
-                    module = self.config._conftest.importconftest(self.fspath)
-                except AttributeError:  # pytest >= 2.8.0
-                    module = self.config.pluginmanager._importconftest(self.fspath)
+                module = self.config.pluginmanager._importconftest(self.fspath)
             else:
                 try:
                     module = self.fspath.pyimport()
-                    # Just ignore searching modules that can't be imported when
-                    # collecting doctests
                 except ImportError:
-                    return
+                    if self.config.getvalue("doctest_ignore_import_errors"):
+                        pytest.skip("unable to import module %r" % self.fspath)
+                    else:
+                        raise
```

Comment:
This changes dramatically the behavior of the plugin.  Now if any module in the repo using this plugin has import failures, it will cause a collection error.  In the past for this plugin, this was ignored.  I see that using `--doctest-ignore-import-errors` can turn this off, but it would be good to document this in the README and the release notes.

Reply:
See https://github.com/astropy/astropy/pull/10242 for a possible fix in astropy btw, we should discuss there whether that is the correct fix.

Reply:
Hmm, sorry for the breakage. The goal was just to reduce the number of difference with pytest's doctest plugin, because of lot of the code here is copied from there, and reducing the differences should make it easier to maintain... 
But yeah as previously this was ignored by default it is probably easier to fix this here.

Reply:
No worries.  I think it is easy to add `--doctest-ignore-import-errors` to addopts in setup.cfg, it was just surprising.  And it took some time to debug.  Hence the desire of making this clear in the documentation.

Reply:
Also, it was nice finding out we had tumbleweed code laying about that had imports that didn't work.

I agree that having its behavior similar to pytest's `doctest` is good.  The more things that are the same, not only easier to maintain, but the less unexpected behavior for people using the plugin as an extension or replacement of the pytest one.

Reply:
see a few open issues about moving these upstream and consolidate the differences between doctest and doctestplus. The blocking factor is time, and now I also think it's better to wait for the situation at pytest to resolve.

Reply:
also, it's my bad that I didn't pick this up during the review, I suppose I put too much trust in tests and was just too happy when this PR finally had a green status.